readthedocs/builds/models.py

```diff
@@ -132,6 +133,37 @@ def __str__(self):
             ),
         )
 
+    @property
+    def ref(self):
+        if self.slug == STABLE:
+            stable = determine_stable_version(self.project.versions.all())
+            if stable:
+                return stable.slug
```

Comment:
This will only return anything on the stable version, that seems less less than useful?

Reply:
What would it return in case it's not stable?

This method return "where the current version points to" and it only makes sense for `stable` which will point to a specific tag/version.

Reply:
I just mean we should call the function `stable_ref` or something -- it doesn't get the `ref` for any Version.